networkx/algorithms/community/label_propagation.py

```diff
@@ -81,7 +81,7 @@ def asyn_lpa_communities(G, weight=None, seed=None):
             # making the algorithm asynchronous.
             label_freq = Counter()
             if weight is None:
-                label_freq.update([labels[v] for v in adj])
+                label_freq.update(map(labels.get, adj))
```

Comment:
I prefer `label_freq.update(labels[v] for v in adj)` for readability. Does it make a difference for speed?

Reply:
The generator expression is 30% slower than the list comprehension for me, which is in turn 30% slower than the map expression. I agree that the list comprehension is much cleaner, but also think that 30% is a relatively sizeable speed-up.

Reply:
I am also surprised that the generator is slower than the list comprehension.
That might depend on the Python version. I'm using 3.8
In any case, the `map` version is indeed faster.  I don' t get 30% faster. 
But I'm testing the whole function, not just the update line.

Thanks!